jmxfetch.py

```diff
@@ -460,6 +451,40 @@ def _get_path_to_jmxfetch(self):
         return os.path.realpath(os.path.join(os.path.abspath(__file__), "..", "..",
                                 "jmxfetch", JMX_FETCH_JAR_NAME))
 
+def get_jmx_checks(auto_conf=True, confd_path=None):
```

Comment:
Not a bad idea to maintain consistency with the fact that service discovery came after the use of `confd_path`.